cdap-ui/app/directives/widget-container/widget-sql-conditions/widget-sql-conditions.js

```diff
@@ -82,6 +79,10 @@ function SqlConditionsController() {
         return field.name;
```

Comment:
There seems to be `JSON.parse` in `initializeOptions` . We should probably wrap that in a `try...catch` block